### PR TITLE
#1324

### PR DESCRIPTION
**Pull request by shroffk**

### CODE REVIEW

File: app/display/representation-javafx/src/main/java/org/csstudio/display/builder/representation/javafx/MacrosTable.java [hunk 1]
```diff
@@ -253,8 +296,8 @@ private void fixup(final int changed_row)
         // Assert one empty row at bottom
         final int len  = data.size();
         if (len <= 0  ||
-            (data.get(len-1).getName().trim().length() > 0  &&
-             data.get(len-1).getValue().trim().length() > 0) )
+                (data.get(len-1).getName().trim().length() > 0  &&
+                        data.get(len-1).getValue().trim().length() > 0) )
             data.add(new MacroItem("", ""));
```
Comment (kasemir) [inline app/display/representation-javafx/src/main/java/org/csstudio/display/builder/representation/javafx/MacrosTable.java:301]: I was actually proud of this formatting, how the data.get(.. lined up for the getName and getValue, but whatever...

Comment (shroffk) [reply app/display/representation-javafx/src/main/java/org/csstudio/display/builder/representation/javafx/MacrosTable.java:301]: Sorry, as you can see I have set the brackets to follow your format. I shall fix this.

Comment (kasemir) [reply app/display/representation-javafx/src/main/java/org/csstudio/display/builder/representation/javafx/MacrosTable.java:301]: These lines are a good example for auto-format simply not understanding what belongs together.

As for all the rest, the whole "<enter name>" stuff could also be removed, but now that you already put in the time to clear it, go ahead.

Comment (shroffk) [reply app/display/representation-javafx/src/main/java/org/csstudio/display/builder/representation/javafx/MacrosTable.java]: For many, directly clicking on the table is easier than the button... do you see any errors in the code

Comment (kasemir) [reply app/display/representation-javafx/src/main/java/org/csstudio/display/builder/representation/javafx/MacrosTable.java]: Hah, web page, can't just enter 'less than'. ...
Meant to say the whole "enter name" stuff .. could be removed, would be less code overall.
But again, go ahead, I've also often ended up with that 'hint' text in the name or value.

Comment (shroffk) [reply app/display/representation-javafx/src/main/java/org/csstudio/display/builder/representation/javafx/MacrosTable.java]: thanks

**Pull request merged**